pkg/operator/controller/route-metrics/controller.go

```diff
@@ -3,6 +3,7 @@ package routemetrics
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/cluster-ingress-operator/pkg/util/ingresscontroller"
```

Comment:
This import can be grouped with the `logf` import.

Reply:
Now the `"golang.org/x/time/rate"` got moved.  * grin *.  Can you put `"golang.org/x/time/rate"` back where it was?

Reply:
Sorry I moved it cause it didn't seem related to openshift packages, but moved it back. Not sure what the organization is supposed to be.

Reply:
Fair enough.  I don't know where this is clearly documented, and we are inconsistent with the ordering of imports across the files in this repository (and other repositories).  We usually group imports into the following groups and ordering:

* Standard packages (e.g. "context", "fmt", "time").
* General non-standard packages (e.g. `"github.com/davecgh/go-spew/spew"`, `"github.com/google/go-cmp/cmp"`,  `"golang.org/x/time/whatever"`).
* OpenShift packages (e.g. `"github.com/openshift/api/operator/v1"`, `"github.com/openshift/library-go/pkg/crypto"`, `"github.com/openshift/cluster-ingress-operator/whatever"`), except ofttimes we put `github.com/openshift/cluster-ingress-operator/*` in its own group, usually before or after other OpenShift packages and in a couple cases as the last group of imports.  
* Kubernetes packages (e.g. `"k8s.io/api/core/v1"`, `"k8s.io/apimachinery/pkg/api/errors"`), except sometimes we put `k8s.io/api/*` and `k8s.io/apimachinery/*` in separate groups, and we usually put `sigs.k8s.io/controller-runtime/*` in another group after that.  

So the basic pattern is typically this: standard packages, non-standard, `github.com/openshift/*`, `k8s.io/*`, `sigs.k8s.io/*`.  However, we have a lot of exceptions, and we have a lot of imports that just don't follow any rules (I often see people do weird things with their imports, some editors screw with import ordering, and I often let it go during reviews).  

This closest thing I have found to an upstream guideline is this obscure GitHub comment: https://github.com/kubernetes/kubernetes/pull/103981#issuecomment-898803440.  One interesting thing about that comment is that it puts imports for the local repository last.  We put `github.com/openshift/cluster-ingress-operator/*` imports last in only a few files in this repository.  

Anyway, if I'm being picky, `"golang.org/x/time/rate"` should be moved to a new group.  However, that import was already grouped with the `github.com/openshift/*` imports, so I prefer the commit that fixes the bug just leave it where it was originally so as not to distract from the important changes in the commit.